src/test/java/it/gov/pagopa/debtposition/controller/DebtPositionControllerTest.java

```diff
@@ -522,7 +522,7 @@ void getDebtPositionListByPaymentDate() throws Exception {
 							.contentType(MediaType.APPLICATION_JSON)).andExpect(status().isOk());
 
 		// effettuo la notifica di pagamento
-		mvc.perform(post("/organizations/LIST_12345678902/paymentoptions/123456IUVMOCK1/pay")
+		mvc.perform(post("/organizations/LIST_12345678902/paymentoptions/"+auxDigit+"123456IUVMOCK1/pay")
```

Comment:
I'd leave a comment, add auxdigit but in general nav and iuv may not have relationships with each other.

Reply:
I have done mixed cases. Sometimes where the nav is custom, other times where it is not passed in the request and therefore you will have: <AUX_DIGIT> + iuv

Reply:
In the pay I chose the path of <AUX_DIGIT> + iuv for speed